### PR TITLE
Added Quality Standards guide on README.md

### PR DESCRIPTION
**Pull request by y27choi**
### Linked issue(s):
Fixes KOL-4390

### What change does this PR introduce and why?
Added QS guide to README.md
Updated metrics for better Debugger experience

### Please check if the PR fulfills these requirements

- [ ] Include reference to internal ticket and/or GitHub issue "Fixes #NNNN" (if applicable)
- [ ] Relevant tests for the changes have been added
- [ ] Relevant docs have been added / updated

### CODE REVIEW

File: examples/dataset/rain_forecast/README.md [hunk 1]
```diff
@@ -38,3 +38,31 @@ optional arguments:
...
+4. \# FP
+5. \# FN
+
+
+### Plots
+1. Confusion Matrix: datapoint.RainTomorrow vs. result.will_rain
+2. Location vs. mean(result.is_FN)
+3. Location vs. mean(result.is_FP)
+4. Year vs. mean(result.is_FN)
+5. Year vs. mean(result.is_FP)
```
Comment (gordonhart) [inline examples/dataset/rain_forecast/README.md]: Nit: monospace code-style things like `mean(result.is_FN)`

File: examples/dataset/rain_forecast/README.md [hunk 2]
```diff
@@ -38,3 +38,31 @@ optional arguments:
...
+
+Once the dataset and results have been uploaded to Kolena, visit [Kolena](https://app.kolena.io/redirect/) to
+test the rain forecast models. See our [QuickStart](https://docs.kolena.io/dataset/quickstart/) guide
+for details.
+
+Here are our [Quality Standards](https://docs.kolena.io/dataset/core-concepts/quality-standard/) recommendations for
+this workflow:
+
+### Metrics
+1. Precision
```
Comment (ly-kolena) [inline examples/dataset/rain_forecast/README.md]: Do we support this in QS?

Comment (gordonhart) [reply examples/dataset/rain_forecast/README.md]: We do! Example on [trunk/yoohee](https://trunk.kolena.io/yoohee/dataset/standards?models=N4IglgJiBcDsBM8A0IDGB7AdgMzAcwFcAnAQwBcwsZRIYBmAXwaA&aggregations=N4Ighg5hBOCmFgC4HtogFwgA5wMYEsBnfZAOxABoQALfCagGzusQ1FvtmgElCAhWIkRcMiaAFdYAXyoMwAI1gMMIAAp4iJclSxhoYALaE2IAGZYVRAPoAxVZRCILmawBV7MkIWTjouWCpwhOIMrFJAA&aggregations=N4Ighg5hBOCmFgC4HtogFwjgYzAGzxABoQALASwlL0tMQ1AqtmgEkBnAIVkURY0TQArrAC%2BJPGABGsQpgBKsXAWIgADmGhgAtuwYgAZgDsMIcuwD6AMQByqxGtPmLAFQAKIcSHbIh0bLCmcOxCePSiQA&aggregations=N4Ighg5hBOCmFgC4HtogFwgGYEYD6AzgMaqwgA0IAFgJYRUA2dViGot9s0AkgQEKxEiLhkTQArrAC%2BlBmABGsBhhAAxHAAIAyiTgUQABzDQwAWwJtsAOxU0CeVQDl9WA7fuqACvsRvMdvAAVbxkQAmRxaCIyTDgCcQZWKSA&aggregations=N4Ighg5hBOCmFgC4HtogFzgMZYK7TCwE8QAaEACwEsIKAbGixDUa22aASQGcAhWRIg4ZE0XLAC%2B5OmABGsOhhABBHPkIlyABzAEAttxYgAZgDslVbgH0AYgDkyJrRes2ACo8TnMlqwBUHckRnH2s-DykQbmR8LFglOG5cOmYJIA&aggregations=N4Ighg5hBOCmFgC4HtogFwgLazAOxABoQAbMAI1hI21zwAo4BnAVxMQDoBhAC3wGNYAeQBmAJTABLPAEoiIAA5hoYLEwygA1rACeNXgOHipBAL7EmyFtEE1mbRCFNA&datasetId=15&qualityStandardId=10)

**Review by munkyshi — APPROVED** [review]

File: examples/dataset/rain_forecast/README.md [hunk 3]
```diff
@@ -38,3 +38,30 @@ optional arguments:
   --model {ann,logreg}  Name of model to test.
   --dataset DATASET     Name of dataset to use for testing.
 ```
+
+## Quality Standards Guide
+
+Once the dataset and results have been uploaded to Kolena, visit [Kolena](https://app.kolena.io/redirect/) to
+test the rain forecast models. See our [QuickStart](https://docs.kolena.io/dataset/quickstart/) guide
```
Comment (ly-kolena) [inline examples/dataset/rain_forecast/README.md:45]: What do you mean by "to test the models"?

Comment (y27choi) [reply examples/dataset/rain_forecast/README.md:45]: test models on the set quality standards — happy to change if you have a suggestion.

**Pull request merged**